elements/pfe-primary-detail/src/pfe-primary-detail.scss

```diff
@@ -94,3 +94,28 @@ $LOCAL-VARIABLES: (
     visibility: visible!important;
   }
 }
+
+::slotted([aria-hidden='true']) {
+  // show detail to visual users but temporarily hide detail from screen readers
+  display: block!important;
+}
+
+::slotted([slot="details-nav--footer"]) {
+  border-right: pfe-local(Border) !important;
+}
+
+// @todo: add note in readme with focus indicator example and show ppl that they need to add a good focus indicator to focusable elements
+::slotted(.focus-styles:focus),
+::slotted(.focus-styles:hover) {
+  outline: 1px dashed #000 !important;
```

Comment:
Where are these values coming from?  Typically our focus styles are link blue solid lines.  Can you share the mock-ups indicating we should be using black dashed line?

Reply:
@castastrophe those styles are coming from the current focus style indicator design for pfe-nav. However, there is a larger discussion taking place about focus indicators. We need to add consistency with them and make them look nice and prominent so Jason is looking into focus styles designs, so they are still tbd. But for primary-detail I matched it with pfe-nav for now since they will be used together. This is the site I pointed out to Jason that I think has awesome focus indicator styles. https://www.smashingmagazine.com/ Would love to hear your thoughts on them!

Reply:
I agree, consistency is important; let's sync up soon and figure out how we can get focus states the same in nav and on page.